cookbooks/aws-parallelcluster-config/templates/default/init/fetch_and_run.erb

```diff
@@ -33,6 +37,13 @@ function download_run (){
     $tmpfile "$@" || error_exit "Failed to run ${ACTION}, ${file} failed with non 0 return code: $?"
 }
 
+function get_stack_status () {
+  region=$(grep cfn_region /etc/parallelcluster/cfnconfig | sed 's/cfn_region=//' | xargs) || return 1
+  stack_name=$(grep stack_name /etc/parallelcluster/cfnconfig | sed 's/stack_name=//' | xargs) || return 1
```

Comment:
you can compact grep and sed in the same command with:
`sed -n 's/stack_name=//p' | xargs`